XML/XML.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="System.Runtime.Serialization.Json" Version="4.3.0" />
     <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.3.0" />
     <PackageReference Include="System.Runtime.Serialization.Xml" Version="4.3.0" />
+    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.5.1" />
```

Comment:
What was the need for this? The last time this was added to enforce the 949 - korean code page it caused massive threading issues.

Reply:
Errors in metafiles, but we could try removing it and going back to normal? This is the .net core equivalent.